nomad/leader.go

```diff
@@ -532,8 +532,12 @@ func (s *Server) reapFailedEvaluations(stopCh chan struct{}) {
 			// due to the fairly large backoff.
 			followupEvalWait := s.config.EvalFailedFollowupBaselineDelay +
 				time.Duration(rand.Int63n(int64(s.config.EvalFailedFollowupDelayRange)))
+
+			// TODO: what is this followup eval and where to create timestamp?
```

Comment:
You would need to modify `CreateFailedFollowupEval` to include create and modify time stamps. 

Failed follow ups are created for failed evaluations, these are new pending evals that point back to the eval that failed.